src/main/java/com/github/krzysiekjodlowski/ox/board/BoardValidator.java

```diff
@@ -0,0 +1,65 @@
+package com.github.krzysiekjodlowski.ox.board;
+
+import com.github.krzysiekjodlowski.ox.Event;
+import com.github.krzysiekjodlowski.ox.Move;
+import com.github.krzysiekjodlowski.ox.Subscriber;
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+
+/**
+ * Verifies if any player wins after his move.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public class BoardValidator implements Subscriber {
+  private boolean isGameOver = false;
+  private Symbol winner = Symbol.EMPTY;
+  private HorizontalChecker horizontalChecker;
+  private VerticalChecker verticalChecker;
+  private SlantChecker slantChecker;
+  private ReverseSlantChecker reverseSlantChecker;
+  private Board board;
```

Comment:
You have a lot of fields in one class. It is usually not a good sign. So can check if there is possiblility to change that somehow (I can see four checkers here so I guess they have to stay but you can try to remove isGameOver, winner and board)

Reply:
isGameOver and winner will be kept in the future Result class, probably on next iteration;)